setup.py

```diff
@@ -1,13 +1,12 @@
 import os
-import re
 import sys
 from setuptools import setup, find_packages
-
+from importlib.metadata import version
 
 PY_VER = sys.version_info
 
-if not PY_VER >= (3, 5):
```

Comment:
I'd just rely on `python_requires` in meta instead, which is needed anyway to get rid of `setup.py`.